src/services/iasql.ts

```diff
@@ -285,7 +285,10 @@ export async function dump(dbId: string, dataOnly: boolean) {
   const dbMeta = await MetadataRepo.getDbById(dbId);
   if (dbMeta?.upgrading) throw new Error('Currently upgrading, cannot dump this database');
   const pgUrl = dbMan.ourPgUrl(dbId);
-  const excludedDataTables = "--exclude-table-data 'aws_account' --exclude-table-data 'iasql_*'";
```

Comment:
if we support snapshotting it might make sense get rid of export since we are not encouraging migrations systems anymore